osu.Android/AndroidManifest.xml

```diff
@@ -2,4 +2,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="sh.ppy.osulazer" android:installLocation="auto">
 	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31" />
 	<application android:allowBackup="true" android:supportsRtl="true" android:label="osu!" android:icon="@drawable/lazer" />
+	<queries>
+		<intent>
+			<action android:name="android.intent.action.VIEW" />
+			<category android:name="android.intent.category.BROWSABLE" />
+			<data android:scheme="https" />
+		</intent>
```

Comment:
The same intent query should be declared for the `mailto:` scheme, as [links of that scheme are also valid for use with `OpenUrlExternally()`](https://github.com/ppy/osu-framework/blob/32a7ef0ceb8eec36f9369a2d8b70cfd0c9c98366/osu.Framework/Extensions/ExtensionMethods.cs#L371-L376). Some places use these sorts of links (most notably wiki). Specifying `mailto:` allows the links to open the mail app to send an email out.

From experimentation, specifying `http` does not seem to be necessary (`https` suffices).